assets/css/v2/triptych/simulation.scss

```diff
@@ -0,0 +1,77 @@
+.simulation-screen-centering-container {
+  display: flex;
+  justify-content: center;
+  background: #3d5366;
+}
+
+.simulation-screen-scrolling-container {
+  scrollbar-color: #607180 #2e3f4d;
+
+  &::-webkit-scrollbar {
+    width: 12px;
+  }
+  &::-webkit-scrollbar-thumb {
+    background-color: #607180;
+    border-radius: 12px;
+    border: 2px solid #2e3f4d;
+  }
+  &::-webkit-scrollbar-track {
+    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+    border-radius: 12px;
+    background-color: #2e3f4d;
+  }
+}
+
+.simulation-screen-scrolling-container {
+  display: flex;
+  overflow-y: hidden;
+  overflow-x: auto;
+  padding-bottom: 20px;
+
+  .simulation-three-flex-panes {
+    height: 344px;
+    width: 580.5px;
+    z-index: 999;
+    display: flex;
```

Comment:
I'd prefer sticking with the usual approach of `position: relative/absolute` + `overflow: hidden` to avoid the possibility of one overflowing widget pushing other content around.

Reply:
I think I made the changes you are suggesting. Let me know if I misunderstood.